src/___new___/data/featureTableComparisonInfo.ts

```diff
@@ -0,0 +1,349 @@
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'N',
+        },
+        {
+          rowHeader: 'Secure Kubernetes nodes with host endpoints managed by Calico',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Apply policy to host-forwarded traffic ',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Dataplane',
+      content: [
+        {
+          rowHeader: 'eBPF',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'iptables',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Windows HNS',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'VPP',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: '',
+          CalicoCloud: '',
+        },
+      ],
+    },
+    {
+      title: 'Images',
+      content: [
+        {
+          rowHeader: 'Scan images for vulnerabilities',
+          CalicoOpenSource: '',
+          CalicoEntreprise: '',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Create policy to block vulnerable images from your clusters',
+          CalicoOpenSource: '',
+          CalicoEntreprise: '',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Observability and troubleshooting',
+      content: [
+        {
+          rowHeader: 'Application level observability and troubleshooting',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Service Graph',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Threat graph',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Elasticsearch  logs (flow, l7, audit, bgp, dns, events)',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Alerts',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Kibana DNS dashboards',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Traffic Flow Visualizer',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Multi-cluster management',
+      content: [
+        {
```

Comment:
Add this item:

{
          rowHeader: 'Multi-cluster management',
          CalicoOpenSource: '',
          CalicoEntreprise: 'Y',
          CalicoCloud: '',
        },

Reply:
Do you mean to have a second item under 'Multi-cluster management' that is also called 'Multi-cluster management'?